### PR TITLE
Fix to retain xlabel when plotting evoked using spatial_colors=True

### PR DESCRIPTION
**Pull request by jdue**
This concerns #4472.

### CODE REVIEW

File: doc/whats_new.rst [hunk 1]
```diff
@@ -148,6 +148,8 @@ BUG
 
     - Fix bug in :func:`mne.viz.plot_compare_evokeds` where ``truncate_yaxis`` was ignored (default is now ``False``), by `Jona Sassenhagen`_
 
+    - Fix bux in :func:`mne.viz.evoked._plot_evoked` where all xlabels were removed when using `spatial_colors=True`, by `Jesper Duemose Nielsen`_
```
Comment (jasmainak) [inline doc/whats_new.rst]: You need to add your webpage at the bottom. Otherwise this won't render correctly

Comment (jdue) [inline doc/whats_new.rst]: okay, how do I do this? right now I have

    .. _Jesper Duemose Nielsen: https://github.com/jdue

at the bottom but that's not it? Sorry I am not so familiar with this.

Comment (agramfort) [inline doc/whats_new.rst]: bux -> bug

File: mne/viz/evoked.py [hunk 2]
```diff
@@ -277,8 +277,9 @@ def _plot_evoked(evoked, picks, exclude, unit, show, ylim, proj, xlim, hline,
                       plot_type=plot_type)
         _draw_proj_checkbox(None, params)
 
-    for ax in fig.axes[:-1]:
-        ax.set_xlabel('')
+    if len(ch_types_used) > 1:
```
Comment (agramfort) [inline mne/viz/evoked.py]: if len(ch_types_used) > 1:
is not needed. Just the:

for ax in fig.axes[:len(ch_types_used)-1]:

File: doc/whats_new.rst (hunk 1 above)
Comment (agramfort) [inline doc/whats_new.rst]: you're good: 

https://5347-1301584-gh.circle-artifacts.com/0/tmp/circle-artifacts.I3K2rym/html/whats_new.html

Comment (agramfort) [inline doc/whats_new.rst]: don't link to mne.viz.evoked._plot_evoked but to:

:func:`mne.viz.plot_evoked`

Comment (jdue) [inline doc/whats_new.rst]: fixed

Comment (jdue) [inline doc/whats_new.rst]: fixed

File: mne/viz/evoked.py (hunk 2 above)
Comment (jdue) [inline mne/viz/evoked.py]: removed the if check

**Comment by agramfort** [comment]
LGTM

File: doc/whats_new.rst [hunk 3]
```diff
@@ -148,6 +148,8 @@ BUG
 
     - Fix bug in :func:`mne.viz.plot_compare_evokeds` where ``truncate_yaxis`` was ignored (default is now ``False``), by `Jona Sassenhagen`_
 
+    - Fix bug in :func:`mne.viz.plot_evoked` where all xlabels were removed when using `spatial_colors=True`, by `Jesper Duemose Nielsen`_
```
Comment (Eric89GXL) [inline doc/whats_new.rst]: double-backticks on code (will fix in `master`)

**Comment by jona-sassenhagen** [comment]
Thanks for fix @jdue !

**Comment by Eric89GXL** [comment]
Merged with a minor cosmit

https://github.com/mne-tools/mne-python/commit/2428dddd397421732d52a735579bd54c5054b39a

**Pull request closed without merging**